@kiva/kv-components/vue/KvTab.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+	<button
+		:id="`kv-tab-${this.for}`"
+		class="tw-text-h3 tw-mb-1.5 tw-whitespace-nowrap"
+		:class="{ 'tw-text-gray-300  hover:tw-text-action-700' : !isActive }"
+		role="tab"
+		:aria-selected="isActive"
+		:aria-controls="`kv-tab-panel-${this.for}`"
+		:tabindex="isActive ? null : -1"
+		@click="handleTabClicked"
+	>
+		<slot></slot>
+	</button>
+</template>
+
+<script>
+export default {
+	inject: ['$KvTabContext'],
+	props: {
+		/**
+		 * A unique id which correspondes to an `id` property on the KvTabPanel it controls
+		 * e.g., <kv-tab for="foo">... <kv-tab-panel id="foo">
+		 * */
+		for: {
+			type: String,
+			required: true,
+		},
+		/**
+		 * The tab should be initially selected.
+		 * */
+		selected: {
+			type: Boolean,
+			default: false,
+		},
+	},
+	computed: {
+		isActive() {
+			const { navItems, selectedIndex } = this.$KvTabContext;
+			return navItems[selectedIndex]?.for === this.for;
+		},
+		index() {
+			const { navItems } = this.$KvTabContext;
+			return navItems?.findIndex((navItem) => navItem.for === this.for);
+		},
+	},
+	mounted() {
+		this.$KvTabContext.navItems.push(this);
```

Comment:
This felt a little funny at first, but seems to work OK. We could also populate the navItems array inside KvTabs.vue by checking `$slots.tabNav` if this seems too weird.

Reply:
I think this seems ok for now. My initial concern was just the extraneous data associated with each instance of a KvTab. This is a fairly light component...In other more complex cases I would def recommend we inject only what is needed.